### PR TITLE
Add Simple Input Example

### PR DESCRIPTION
**Pull request by SteveMaybe**
This PR includes a basic input example demonstrating passing `clientside` values back to the server, and returning a script to update the client.

### CODE REVIEW

File: examples/Input/Input.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func main() {
+
+	Name := &clientside.String{}
+	FullName := &clientside.String{}
+
+	app.New("Input Example",
+		column.New(
+			row.New(
```
Comment (SteveMaybe) [inline examples/Input/Input.go:28]: I wanted to include something like:

```go
seed.If(<FullName  == "">, set.Hidden())
```

here, but I couldn't quite figure out how to create a condition off the clientside value.

Comment (SteveMaybe) [reply examples/Input/Input.go:28]: The following looks like it should work, but doesn't either 🤔
```go
	visible.When(FullName.GetString().Equals(js.NewString("")).Not(), row.New(
		text.New(text.SetStringTo(FullName)),
	)),
```

Comment (Splizard) [reply examples/Input/Input.go:28]: Ah, that's close. All client types implement the client.Bool interface so they are (similar to javascript) can be treated as booleans. So you can achieve the same thing by passing the string directly:
```go 
visible.When(FullName. ...)
```

More complex conditions, can be constructed with the the client/if/* packages.
IE. 
```go
import "qlova.org/seed/client/if/the"
visible.When(the.String(FullName).Equals("SteveMaybe"), ...)
```
or logical and/or not with the all, any & not packages.
```go
import "qlova.org/seed/client/if/any"
visible.When(any.AreTrue(...), ...)
```

You can also change styles and most builtin seed.Option(s) dynamically with the change package.
```go
import "qlova.org/seed/set/change"
change.When(FullName, text.SetSize(rem.New(2))
```

**Comment by Splizard** [comment]
Thanks!

**Pull request merged**